pkg/frontend/mysql_cmd_executor.go

```diff
@@ -68,13 +68,10 @@ func parameterModificationInTxnErrorInfo() string {
 }
 
 var (
-	errorComplicateExprIsNotSupported              = goErrors.New("the complicate expression is not supported")
```

Comment:
Please get rid of all of these.   Use moerr.

Reply:
Done.